stdlib/public/core/SliceBuffer.swift

```diff
@@ -511,3 +511,20 @@ extension _SliceBuffer {
     return ContiguousArray(_buffer: result)
   }
 }
+
+extension _SliceBuffer {
+  @_alwaysEmitIntoClient
+  internal func isTriviallyIdentical(to other: Self) -> Bool {
+    guard
+      // FIXME: use builtin == function
+      // self.owner == other.owner,
+      unsafe unsafeBitCast(self.owner, to: Int.self) == unsafeBitCast(other.owner, to: Int.self),
```

Comment:
I think you'll need to use:

* `==` for Embedded Swift, because `owner` is `Builtin.NativeObject` (via the `_SliceBuffer.AnyObject` typealias).

* `===` for standard Swift, because `owner` is `Swift.AnyObject`.

The [status](https://docs.swift.org/embedded/documentation/embedded/status) page lists `AnyObject` as currently supported in Embedded Swift, so perhaps this can be simplified in the future?